packages/sysdig/data_stream/event/fields/fields.yml

```
@@ -0,0 +1,415 @@
+- name: sysdig
+  type: group
+  fields:
+    - name: event
+      type: group
+      fields:
+        - name: actions
+          type: group
+          fields:
+            - name: after_event_ns
+              type: long
+              description: Amount of nanoseconds after the event the Capture spans.
```

```suggestion
              description: Duration after the event that the capture spans in nanoseconds.
```
(similar changes below)